docs/content/preview/yugabyte-platform/manage-deployments/edit-helm-overrides.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Edit Kubernetes overrides
+headerTitle: Edit Kubernetes overrides
+linkTitle: Edit Kubernetes overrides
+description: Edit Kubernetes overrides for a universe.
+aliases:
+menu:
+  preview_yugabyte-platform:
+    identifier: edit-helm-overrides
```

Comment:
Any reason why the file name and the identifier is `edit-helm-overrides` and not `edit-kubernetes-overrides` ?

Reply:
helm is shorter than kubernetes :)